docs/source/index.rst

```diff
@@ -32,6 +32,7 @@ PyTorch Lightning Documentation
    :caption: Community Examples
 
    Contextual Emotion Detection (DoubleDistilBert) <https://github.com/PyTorchLightning/emotion_transformer>
+   Cotatron- Transcription-Guided Speech Encoder <https://github.com/mindslab-ai/cotatron>
```

Comment:
```suggestion
   Cotatron-Transcription-Guided Speech Encoder <https://github.com/mindslab-ai/cotatron>
```

Reply:
This is how it is written in their github repo:
```suggestion
   Cotatron: Transcription-Guided Speech Encoder <https://github.com/mindslab-ai/cotatron>
```